parsing/dialects/default/config.fmpp

```diff
@@ -16,7 +16,7 @@
 data: {
   parser: {
     # Generated parser implementation class package and name
-    package: "org.apache.calcite.sql.parser.default",
+    package: "org.apache.calcite.sql.parser.defaultparser",
```

Comment:
nit: Why change this from "default" to "defaultparser"?

Reply:
It was throwing an error because "default" is a reserved keyword in Java

Reply:
Ah I see. Will there be confusion if we have the name of the dialect (e.g. "default") differ from the path/package name? Should we instead just name "default" and its corresponding package/path to "DefaultDialect"/"defaultdialect"?

Reply:
Yeah that makes sense. I can change everything to defaultdialect/DefaultDialect so the package and class names match